README.md

```diff
@@ -49,11 +49,18 @@ jobs:
 
 ## Tips for Gradle
 
-Use junit report inside test block of gradle config:
+As Gradle uses a different build directory than Maven by default, you might need to set the `report_paths` variable:
+
+```yaml
+    `report_paths: '**/build/test-results/test/TEST-*.xml'`
```

Comment:
There are some unnecessary backticks around the line

Reply:
Fixed.